pagopa/io-app
# fix(IDPay): [IODPAY-122] Fix instrument enrollment CTA loading state

## Short description
This PR fixes the loading state of the Continue button in the payment method enrollment screen

| Before 	| After 	|
| ---	| ---	|
|   <video src="https://user-images.githubusercontent.com/6160324/230160833-17e7cacf-857b-4eff-b7bd-13010c355670.mp4" /> | 	 <video src="https://user-images.githubusercontent.com/6160324/230160868-0317d690-e0d9-48d4-98d6-452c7813f362.mp4" /> |

## List of changes proposed in this pull request
- InstrumentEnrollmentScreen.tsx
  - Added `isLoading` prop to `Continue` button;
  - Removed label if loading state is `true`

## How to test
- With the local dev server started, go to the wallet section of the app and open an initiative not configured.
- Start the configuration process and reach the instrument enrollment step.
- Check if the continue button has the correct behavior when enrolling an instrument.
